contrib/ci/generate_docker.py

```
@@ -31,75 +24,109 @@ def get_container_cmd():
-        args += [f"--build-arg=http_proxy={os.environ['http_proxy']}"]
-    if "https_proxy" in os.environ:
-        args += [f"--build-arg=https_proxy={os.environ['https_proxy']}"]
-    args += ["-f", "./Dockerfile", "."]
-    subprocess.check_call(args)
+    build_args = [cmd, "build", "-t", f"fwupd-{args.distro}"]
+    if http_proxy := os.environ.get("http_proxy"):
+        build_args += [f"--build-arg=http_proxy={http_proxy}"]
+    if https_proxy := os.environ.get("https_proxy"):
+        build_args += [f"--build-arg=https_proxy={https_proxy}"]
+    build_args += ["-f", "./Dockerfile", "."]
+    subprocess.check_call(build_args)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)